shared/constants/field/myinfo/index.ts

```diff
@@ -17,6 +17,7 @@ export type MyInfoFieldBlock = {
   fieldType: MyInfoField['fieldType']
   fieldOptions?: string[]
   ValidationOptions?: Record<string, unknown>
+  prefilledValue: string
```

Comment:
nit: should we add documentation to what prefilledValue means? Actually should we rename it to `previewValue` instead? make it more clear

Reply:
actually that makes alot of sense - was thinking of it as `prefilled` as the values are supposed to be pre-filled but these will only be seen in the preview ._.